app/views/posts/index.html.erb

```diff
@@ -16,7 +16,7 @@
    <br>
```

Comment:
- [ ] It would be better to add the instructions about how to run the test to our README file :+1: since this is a testing project :smile: